Kudu.Agent/Controllers/KuduProcessController.cs

```diff
@@ -0,0 +1,324 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Kudu.Agent.Contracts;
+using Kudu.Agent.Util;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Authentication;
+using System.Threading;
+
+namespace Kudu.Agent.Controllers
+{
+    [ApiController]
+    [Route("/processes")]
+    public class KuduProcessController : ControllerBase
```

Comment:
this was just an example used for the previous implementation, I was under the impression that the new implementation will be reusing Kudu.Core and Kudu.Services, isn't it?

Reply:
I'm not sure I understand. I was not planning on converting Kudu.Services (where this endpoint lives normally) to .NET 6. Kudu.Core will be converted but it relies on Services for this functionality.